README.rst

```diff
@@ -87,6 +87,22 @@ Or, if you want it without pip-chill::
  pytest==6.2.1
  ...
 
+Or, if you want it formatted for conda::
+
+ $ pip-chill --conda
+ name: myenv
+ channels:
+   - defaults
+dependencies:
```

Comment:
Indentation here seems odd - is this block supposed to be one column to the left?

Reply:
you're right the indentation is a bit off here, but it was intended - just got a bit sloppy.

example from the [conda docs](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#activating-an-environment):
```
name: env-name
channels:
  - conda-forge
  - defaults
dependencies:
  - python=3.7
  - codecov
  ```